src/common/hooks/useOrganization/useOrganization.js

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+
+import { VENDORS_API } from '@folio/stripes-acq-components';
+
+export const useOrganization = (id) => {
+  const ky = useOkapiKy();
+
+  const { isLoading, data } = useQuery(
+    ['ui-organizations', 'organization', id],
```

Comment:
`ui-organizations` ^ -> use `useNamespace` hook instead and mb it's good reason to move this hook to stripes-acq-components to avoid duplications?

Reply:
Yeah, I think it's a good idea, thx